cli.js

```diff
@@ -120,10 +132,10 @@ function generate(filepath, email) {
 
 async function init() {
 	const results = globby.sync([
-		'code_of_conduct.*',
-		'code-of-conduct.*',
-		'.github/code_of_conduct.*',
-		'.github/code-of-conduct.*'
+		`${directory}/code_of_conduct.*`,
+		`${directory}/code-of-conduct.*`,
+		`${directory}/.github/code_of_conduct.*`,
+		`${directory}/.github/code-of-conduct.*`
```

Comment:
Instead of https://github.com/sindresorhus/conduct/pull/14/files#diff-2cce40143051e25f811b56c79d619bf5R96, you can just use `path.posix.join()` here. That ensures forward slashes.